configs/test/jest.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "verbose": true,
-  "notify": true,
+  "notify": false,
   "collectCoverageFrom": ["**/*.{jsx,js,coffee,cjsx}", "!**/node_modules/**"],
```

Comment:
this is still keeping `terminal-notifier` running and gobbling memory when spec's fail 😦